src/com/company/Main.java

```diff
@@ -67,9 +88,16 @@ public static void searchTheWord(String data, String searchword) {
         }
         if (count == 0) {
             System.out.println("Word not found");
+            String errorMessage = "Word not found";
+            database.databaseInsertion(txtFilePath, searchWord, count, status, errorMessage);
+
         } else {
             System.out.println("The word has been found");
             System.out.println("The word has been repeated for " + count + " times");
+            String status1 = "success";
+            String errorMessage = "null";
```

Comment:
There is no need to have line 97 and 98, you can directly add it as a parameter to the method call in 99.

Reply:
yeah that has been done.